### ISSUE TITLE
Cloud data storage support for embedding

### ISSUE THREAD

**Issue by themayakumari** [embedchain/embedchain#573]
### 🚀 The feature

We have pdf and videos stored on Google Firebase, can we have a feature that gets the data from firebase for embedding and get the answers for various types of content like pdf, video, audio content

### Motivation, pitch

This feature will significantly help to reuse the content (video, audio, pdfs etc) already stored on Google Cloud.

**Comment by cachho** [comment]
Does firebase give you a direct link for each file? Is it publicly accessible?

**Comment by themayakumari** [comment]
Yes, we do get direct link of each file. But thinking if we can folder access so that multiple files can be used.ThanksOn Sep 10, 2023, at 3:56 PM, cachho ***@***.***> wrote:﻿
Does firebase give you a direct link for each file? Is it publicly accessible?

**Comment by parshvadaftari** [comment]
Closing as stale.
